08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which is used to pick a single winner from a list of several candidates. 
+%Normally, voters cast a single vote for their preferred candidate, but in our game specification it is also possible for voters to make their preference list.
+Runoff voting is widely recognized and used around the world primarily for the election of the head of state. It is an effective voting method for the selection of candidates.
+
+\texttt{Voter} islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, two candidates with most first placed votes are chosen among all candidates. The Runoff voting can conclude the election in the first-round if one candidate obtains a majority. Otherwise, the second round need to be considered and utilized where the ballots will be recounted by eliminating the candidates other than the top-two. Subsequently, the candidate with the most votes between the top-two candidates will be the winner.
+
+\textbf{Instant Runoff} is a voting method which is used in single-seat elections which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated and this step is repeated until only one candidate remains.
+
+% In the event of tied result, alternative rules can be chosen to give a single winner. It is obvious that the effectiveness of this method comes from its flexibility of various rules available in a tie situation.
+
+\textbf{Approval} is a voting method for single winner in which each voter selects a subset of candidates. As of its procedures and usage in the game, Approval will be used for the elections of roles where each voter picks a group of candidates and the most named or approved candidate wins.
+
+\textbf{Tied situation:} In a tied situation, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes. The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work when preference lists form a cycle, which is known as the Condorcet paradox or voting paradox. Alternatively, Hare Rule can be used in a tiebreak where all candidates with the least first-place votes will be all eliminated. Moreover, Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
```

Comment:
do note say \textbf{Tied situation} -- this NOT  another method lol.

suggested re-writing:
**In case of a tied election,** there are three possible methods that can be used as a tiebreaker.

please `\begin{itemize} \end{itemize}` these three methods

Reply:
changed